generate-isbn-ranges.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+import urllib.request
+import xml.etree.ElementTree as ET
+import sys
+
+# See https://www.isbn-international.org/range_file_generation
+url = 'http://www.isbn-international.org/export_rangemessage.xml'
```

Comment:
Or they did when I was looking at this before. I remember considering HTTP vs HTTPS